server/src/main/java/org/elasticsearch/http/DefaultRestChannel.java

```diff
@@ -77,6 +84,9 @@ protected BytesStreamOutput newBytesOutput() {
 
     @Override
     public void sendResponse(RestResponse restResponse) {
+        if (tracerLog != null) {
+            tracerLog.trace(new ParameterizedMessage("[{}] sent response to [{}]", request.hashCode(), httpChannel));
```

Comment:
let's also include `restResponse.status()` here, as well as content type, length and X_OPAQUE_ID

Reply:
Should we move this to the finally block where we have actually sent the response?

This allows us to log something different in case where sending the response failed.